releasenotes/notes/add-manila-share-types-12fa414e15ef0b6b.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - Add support for Manila share types configuration with default extra
+    specs for snapshot support.
```

Comment:
would be nice to mention a default public share type will also be created

Reply:
Default share type is created by chart during bootstrap https://github.com/vexxhost/atmosphere/blob/main/charts/manila/templates/bin/_bootstrap.sh.tpl#L54
and we rely on it by default also: https://github.com/vexxhost/atmosphere/blob/main/charts/manila/values.yaml#L651